app/assets/stylesheets/common/components/keyboard_shortcuts.scss

```diff
@@ -74,8 +82,11 @@
     li {
       align-items: center;
       display: flex;
-      flex-wrap: wrap;
-      margin-bottom: 6px;
+      flex-wrap: nowrap;
+      flex-direction: row-reverse;
```

Comment:
That's ok as a first iteration, but ideally we would write the proper html

Reply:
Yeah, I see what @jjaffeux means here, using only CSS to reverse the layout of the items has some downsides. Screen readers will read it as before and not sure if this works as is for RTL locales. 

Fixing this properly is also tricky though, the shortcut part and the description part are currently both in the same locale string: 

```
notifications: "%{shortcut} Open notifications"
hamburger_menu: "%{shortcut} Open hamburger menu"
user_profile_menu: "%{shortcut} Open user menu"
```

Maybe the right thing to do here is to stop including the `%{shortcut}` in the locale string?

Reply:
Thanks @jjaffeux ! Will refactor in the next PR.